packages/cactus-plugin-bungee/src/test/typescript/fabric-contracts/simple-asset/chaincode-typescript/tslint.json

```diff
@@ -0,0 +1,23 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "jsRules": {},
+    "rules": {
+        "indent": [true, "spaces", 4],
+        "linebreak-style": [true, "LF"],
+        "quotemark": [true, "single"],
+        "semicolon": [true, "always"],
+        "no-console": false,
+        "curly": true,
+        "triple-equals": true,
+        "no-string-throw": true,
+        "no-var-keyword": true,
+        "no-trailing-whitespace": true,
+        "object-literal-key-quotes": [true, "as-needed"],
+        "object-literal-sort-keys": false,
+        "max-line-length": false
+    },
+    "rulesDirectory": []
+}
```

Comment:
@eduv09 I'm guessing that this file can be removed because it won't be used at all in the test automation context. Please either remove it or just advise why it is necessary and if there is a legit reason for it to remain then I'm totally OK with that too we just have to have it documented on the purpose of leaving it under version control.